projects/step-frontend/src/lib/modules/scheduler/execution-task-parameters-list/execution-task-parameters-list.component.ts

```diff
@@ -0,0 +1,141 @@
+import { Component, OnDestroy } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { downgradeComponent, getAngularJSGlobal } from '@angular/upgrade/static';
+import {
+  AJS_MODULE,
+  DialogsService,
+  IsUsedByDialogsService,
+  AuthService,
+  ContextService,
+  Mutable,
+  SchedulerService,
+  TableRestService,
+  ExecutiontTaskParameters,
+  TableLocalDataSource,
+} from '@exense/step-core';
+import { BehaviorSubject, switchMap, of, catchError, noop, shareReplay, tap, map, lastValueFrom } from 'rxjs';
+import { ExecutionTaskParameterDialogsService } from '../services/execution-task-parameter-dialogs.service';
+import { Location } from '@angular/common';
+import { DashboardService } from '../../_common/services/dashboard.service';
+
+type InProgress = Mutable<Pick<ExecutionTaskParametersListComponent, 'inProgress'>>;
+
+@Component({
+  selector: 'step-execution-task-parameters-list',
+  templateUrl: './execution-task-parameters-list.component.html',
+  styleUrls: ['./execution-task-parameters-list.component.scss'],
+})
+export class ExecutionTaskParametersListComponent {
+  readonly STATUS_ACTIVE_STRING = 'On';
+  readonly STATUS_INACTIVE_STRING = 'Off';
+
+  readonly currentUserName: string;
+  readonly inProgress: boolean = false;
+
+  private executionTaskParametersRequest$ = new BehaviorSubject<any>({});
+  private executionTaskParameters$ = this.executionTaskParametersRequest$.pipe(
+    tap((_) => ((this as InProgress).inProgress = true)),
+    switchMap((_) => this._schedulerService.getScheduledExecutions()),
+    tap((_) => ((this as InProgress).inProgress = false)),
+    shareReplay(1)
+  );
```

Comment:
@chillbirdo 
I guess we use this to solve the reload issue, right?

I just discovered we have that one in all migrated tables, I don't like this code though. Can you check if there is a solution to nicely capsulate this in a service / inheritedServiceClass so we only call "service.reloadData()" or something in that direction?

Reply:
Absolutely, I was thinking the same, I actually did a note to myself here: https://github.com/exense/step-frontend/pull/41#issuecomment-1168715952